### PR TITLE
Deploy contracts for 5 new data feeds

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

includes deployment infromation for both `cypress` and `baobab`

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: contracts/migration/baobab/Aggregator/migration.lock [hunk 1]
```diff
@@ -59,3 +59,9 @@
 20231102141119_KSP-KRW.json
 20231102141119_MBX-KRW.json
 20231102141119_WEMIX-USDT.json
+20231114131186_CHF-USD.json
+20231114131186_EUR-USD.json
+20231114131186_GBP-USD.json
+20231114131186_JPY-USD.json
+20231114131186_KRW-USD.json
+accountList.json
```
Comment (martinkersner) [inline contracts/migration/baobab/Aggregator/migration.lock:67]: Please remove the `accountList.json`.

File: contracts/migration/cypress/Aggregator/migration.lock [hunk 2]
```diff
@@ -47,3 +47,9 @@
 20231102141113_KSP-KRW.json
 20231102141113_MBX-KRW.json
 20231102141113_WEMIX-USDT.json
+20231114131165_CHF-USD.json
+20231114131165_EUR-USD.json
+20231114131165_GBP-USD.json
+20231114131165_JPY-USD.json
+20231114131165_KRW-USD.json
+accountList.json
```
Comment (martinkersner) [inline contracts/migration/cypress/Aggregator/migration.lock:55]: Please remove the `accountList.json`.

File: contracts/scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs [hunk 3]
```diff
@@ -1,29 +1,16 @@
...
-  'DOT-USDT',
-  'ETH-USDT',
-  'KLAY-USDT',
-  'MATIC-USDT',
-  'SOL-USDT',
-  'TRX-USDT',
-  'USDC-USDT',
-  'MNR-KRW'
-]
+const priceFeeds = ['CHF-USD', 'EUR-USD', 'GBP-USD', 'JPY-USD', 'KRW-USD']
```
Comment (martinkersner) [inline contracts/scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs:4]: I think we should not modify this file if we use it. In the mid-term, we should pass names of price feeds as parameters. Please make an issue for it 🙏

**Review by martinkersner — APPROVED** [review]
LGTM! There are just few minor issues. Please check comments before merging.

**Review by bayram98 — APPROVED** [review]
LGTM!

File: contracts/migration/baobab/Aggregator/20231114131186_CHF-USD.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	"changeOracles": {
+		"removed": [],
+		"added": [
+			"0xa68a486a654Ca02623e8A41f703A1b24A2cd666e"
+		],
+		"minSubmissionCount": 1,
+		"maxSubmissionCount": 1,
+		"restartDelay": 0
+	}
+}
```
Comment (bayram98) [inline contracts/migration/baobab/Aggregator/20231114131186_CHF-USD.json:18]: @nick-bisonai just small comment, I think you used different tab formatting that our general where we use `tabWidth` with 2 space. Maybe worthy to check it for next time so we follow the same formatting.  :)

File: contracts/scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs (hunk 3 above)
Comment (nick-bisonai) [reply contracts/scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs]: yes I have made an issue, thank you

https://github.com/Bisonai/orakl/issues/925

**Pull request merged**